src/services/measure.service.js

```diff
@@ -337,7 +337,7 @@ const evaluateMeasureForIndividual = async (args, { req }) => {
     measurementPeriodEnd: req.query.periodEnd
   });
 
-  const { periodStart, periodEnd, reportType = 'individual', subject, practitioner } = req.query;
```

Comment:
This made me investigate default report types a little bit, and it should probably be addressed higher in this file (in the main `evaluateMeasure` handling?), but we should be following these defaults:
`If not specified, a default value of subject will be used if the subject parameter is supplied, otherwise, population will be used`

Reply:
Good catch. Just pushed a fix. New logic is:

1. If BOTH reportType AND subject are missing, use population
2. If reportType is missing and subject is NOT missing, use subject

This adheres to the spec as I understand it now